Provide more information regarding constructing the admin consent URL. (#111454)
Azure/azure-content: articles/active-directory/manage-apps/grant-admin-consent.md

where:

- `{client-id}` is the application's client ID (also known as app ID).
- `{organization}` is the tenant ID or any verified domain name of the tenant you want to consent the application in. You can use the value `organizations`, which will cause the consent to happen in the home tenant of the user you sign in with.

As always, carefully review the permissions an application requests before granting consent.

> For more information regarding constructing the tenant-wide admin consent URL, see [Grant tenant-wide admin consent to an application](../develop/v2-admin-consent.md)

Reviewer: ```suggestion
For more information on constructing the tenant-wide admin consent URL, see [Admin consent on the Microsoft identity platform](../develop/v2-admin-consent.md).
```

Reviewer: I think the wording here needs to reflect what the article you're linking to explains. Feel free to sign-off after considering the suggestion. Thanks
